src/adapters/feature-flags.ts

```diff
@@ -0,0 +1,81 @@
+import { IBaseComponent, START_COMPONENT, STOP_COMPONENT } from '@well-known-components/interfaces'
+import { AppComponents } from '../types'
+import { ApplicationName } from '@well-known-components/features-component'
+
+export enum FeatureFlag {
+  /**
+   * Feature flag to enable/disable AI compliance for communities.
+   *
+   * @example
+   * Variant value: "true" or "false"
```

Comment:
NIT: it won't have values on its variant, it will be just on or off

In FF terms, we are using `variant` concept to state that the FF besides being on or off it will also contain values

Reply:
I left the old comment and only changed the values, nice catch, thanks!